src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -29,40 +29,35 @@ public void equals_False_X10() {
   }
 
   @Test
-  public void equals_IsReflexive_BHamrick() {
+  public void equals_True_Reflexive() {
     assertTrue(rnX.equals(rnX));
   }
 
-  public void equals_IsTransitive_BHamrick() {
+  public void equals_True_Transitive() {
     RomanNumeral x = new RomanNumeral("IV");
     RomanNumeral y = new RomanNumeral("IV");
     RomanNumeral z = new RomanNumeral("IV");
 
-    assertTrue(x.equals(y) | y.equals(z) | x.equals(z));
+    assertTrue(x.equals(y) && y.equals(z) && x.equals(z));
   }
 
   @Test
-  public void equals_IsSymmetric_BHamrick() {
+  public void equals_True_Symmetric() {
     RomanNumeral x = new RomanNumeral("IV");
     RomanNumeral y = new RomanNumeral("IV");
 
-    assertTrue(x.equals(y) | y.equals(x));
+    assertTrue(x.equals(y) && y.equals(x));
   }
 
   @Test
-  public void equals_IsStableOverTime_BHamrick() {
+  public void equals_True_StableOverTime() {
```

Comment:
As mentioned, I don't think this can be tested for.

Reply:
Could you please remove this test?

Reply:
Yes! Removed!